misc/travis-install-bashate.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# This script is public domain.
+
+# This script installs bashate by unpacking .deb files built for Ubuntu 18.04.
+# This is intended to install/use bashate in TravisCI.
+
+# Usage (.travis.yml):
+#   ...
+#   os:
+#     - linux
+#     - ...
+#   sudo: required
+#   dist: trusty
+#   ...
+#   addons:
+#     apt:
+#       packages:
+#       - python-tz
+#       - ...
+#   ...
+#   before_install:
+#       - time sh ./misc/travis-install-bashate.sh
+#       - ...
+
+set -x
```

Comment:
You probably want (at least) set -e and set -u as well.

Reply:
`set -x` was replaced with `set -eux`, and `if ... fi` for `curl` was removed.